dif-endorsed/README.md

```diff
@@ -0,0 +1,49 @@
+| `did:webplus`       | ?                                                                                                               | no                                     | [yes](https://github.com/decentralized-identity/did-methods/blob/main/method-proposals/PROPOSAL-did-webplus.md) | no                                                                                                                                  | no                                      | no                           | 0                                                                                                               | ?                                         | ?                             | ?                           | **no**                       |
+| `did:webs`          | [Implementors Draft v0.9.15](https://trustoverip.github.io/tswg-did-method-webs-specification/)                 | no                                     | [yes](https://github.com/decentralized-identity/did-methods/blob/main/method-proposals/PROPOSAL-did-webs.md)    | no                                                                                                                                  | yes                                     | no                           | 0                                                                                                               | ?                                         | ?                             | ?                           | **no**                       |
+| `did:webvh`         | [Current Stable v0.5](https://identity.foundation/didwebvh/)                                                    | yes                                    | [yes](https://github.com/decentralized-identity/did-methods/blob/main/method-proposals/PROPOSAL-did-webvh.md)   | no                                                                                                                                  | yes                                     | yes                          | 0                                                                                                               | ?                                         | ?                             | ?                           | **no**                       |
+
+(1) Use alphabetical order.
+
+(2) Must be some kind of final version, include link to specification.
```

Comment:
For the `last updated <date>` items, it would be good to link to a similarly datestamped version of the document.